proposals/description/fip_99.ts

```diff
@@ -0,0 +1,53 @@
+import { ProposalDescription } from '@custom-types/types';
+
+const fip_99: ProposalDescription = {
+  title: 'FIP-99: Sell RAI',
+  commands: [
+    {
+      target: 'ratioPCVControllerV2',
+      values: '0',
+      method: 'withdrawRatio(address, address, uint256)',
+      arguments: ['{rariPool9RaiPCVDeposit}', '{aaveRaiPCVDeposit}', '8000'],
+      description: 'Withdraw all Rai from the rari pool 9 pcv deposit and sent to the aave rai pcv deposit'
+    },
+    {
+      target: 'aaveRaiPCVDeposit',
+      values: '0',
+      method: 'deposit()',
+      arguments: [],
+      description: 'Call deposit() on the aave rai pcv deposit'
+    },
+    {
+      target: 'core',
+      values: '0',
+      method: 'grantMinter(address)',
+      arguments: ['{raiPriceBoundPSM}'],
```

Comment:
The RaiPriceBoundPSM does not need the minter role as it will only be selling RAI in exchange for Fei.

Reply:
We might want to enable minting in the future

Reply:
The forum post literally says sell only, so I would stick with that https://tribe.fei.money/t/fip-99-sell-rai/4120